### ISSUE TITLE
Replace JSON data format

### ISSUE THREAD

**Issue by frankhale** [frankhale/toby#63]
In the rewrite (which has not been committed yet) I've replaced the JSON data file format with a custom format with a very clean syntax. Gone are the image thumbnails because the URL to them is universal and easily inferred.

The format is:

```
group_name {
  video title here : youtube-id
}
```

Here is what my current data file looks like:

```
EDM Sets {
  Trance mix 2014 - Armin Van BuurenW&WPaul Oakenfold : p68MTskBUnM
  Progressive Trance Top 10 (February 2014) / New Trance Mix / Paradise : g2L0OWterVg
  Goodbye 2013 Mix (Progressive Trance House Trouse Uplifting Melodic Vocal) : RLVs7Csdmf8
  Best of Progressive & Melodic Trance 2013 Yearmix [5 Hour Set] [HD] - Shane Collins - APJ #32 : seBS2MEtALE
  Arctic Moon Megamix - best tracks & remixes : aX00On-X7K8
  Seven Lions Mix (Hour Long Melodic Dubstep / Trancestep Mix) : 7KfKft2lcog
  BEST OF VOCAL TRANCE | MIX#4 : VZT9HGsv01A
  This Was The Summer Of Hardstyle 2013 : kQko_qnKoW4
  Incredible Emotional Vocal Trance Mix l April 2015 (Vol 26) : 3l2FHxVd71o
  Dash Berlin - The Official Video Hit Mix : U6l9NdAJwRk
  Dash Berlin - Live @ Ultra Music Festival Miami Mainstage 2015 (Full Set) : tS1NCigEB_I
}

EDM {
  Dash Berlin - Underneath The Sky (Sunsound Chillout Remix) : UEqMD-5urik
  Jasper Forks - River Flows In You (Out of Blackout Vocal Edit) [HD] : 5UwnhliP5N8
  Yuri Kane - Right back : G9AG0wkLIp0
  Dash Berlin feat Jonathan Mendelsohn - Better Half Of Me (Acoustic) : De3Cjo23dOs
  Kyau & Albert - Another Time [Official Video] + Lyrics : zlBo93AqZmg
  R3hab & NERVO & Ummet Ozcan - Revolution (Vocal Mix) : 92P57Uk1VbM
  Gareth Emery feat Roxanne Emery - Soldier : 7b1WeyQ5cHw
  Paul van Dyk feat Adam Young - Eternity (Official Music Video) : FzT9YppKeeQ
  BT feat Jes - Every other way (armin mix) : cTXdgzb4wkU
  New World - Ushio (Emotional Intro Mix) [As Played on Uplifting Only 090] : gfAzQDCrUnQ
  Gareth Emery - Long Way Home [Official Video] : 0bj4i-sW44s
  Dreamy - Fusion (New World Remix) [Diverted Music Promo] : D_8y-XX9wM8
  Sunbrothers - Everything (Original Mix) : wrMBe5cyNKA
  Somna & Jennifer Rene - Because You're Here (Sunset Remixi) : 8hERMvja-fA
  Dash Berlin & Disfunktion feat Chris Arnott - People Of The Night : b4ZuXTiWX44
  Morgan Page Sultan + Ned Shepard & BT - In the Air feat Angela McCluskey : WyK_82zrfTY
  Afrojack - Ten Feet Tall (Lyric Video) ft Wrabel : bltr_Dsk5EY
  Gareth Emery feat Roxanne Emery - Too Dark Tonight : WZFgVGU7pFk
  Arcane Science feat Melissa Loretta - Still Feel (You Here) (Intro Mix) : v9zmvPSHj9I
  Beat Service & Ana Criado - So Much Of Me Is You (Original Mix) : an_FDsq1Pkk
  Estiva & Cardinal feat Arielle Maren - Wait Forever (Daniel Kandi's Bangin Remix) : P0fp7fDxcVA
  AMR feat Ai Takekawa - Beyond The Moon (Original Mix) : VXPor8AsY48
  Moonbeam feat Matvey Emerson - Wanderer (Original Mix) : hT8KPRbzP8g
  Rigby - Earth Meets Water (Official Video) : 8d3fWzPzP4A
  Rigby - Earth Meets Water (Wildstylez Remix) : TyKvwXUy1aU
  Tom Swoon Lush & Simon - Ahead Of Us (Lyric Video) : Btz1lteUtuM
  Skrux ft Delacey - My Love Is A Weapon : oqbmz9ZncEs
  Myon & Shane54 feat Aruna Helpless : mgmPydUzhgg
  illitheas - Epica (Intro Mix) [Abora Skies] Promo [Uplifting Only #119] Video Edit : GxfwZMdSt4w
  Lost Frequencies - Are You With Me (Dash Berlin Remix) (Lyrics Music Video) : KYkRKvHYTls
  Ikerya Project feat Muhib Khan - It's Over Now (Original Mix) [As Played on Uplifting Only 115] : flXZE9izGP0
  Syntouch - Into The Sky (Extended Orchestral Mix) [As Played on Uplifting Only 109] : OuAkh9FqXs0
  Phoebe Ryan - Mine (Illenium Remix) : OHid2FWShAw
  Dash Berlin ft Sarah Howells - Go It Alone (Official Music Video) : IMepLfEJu6w
  Cosmic Gate & Emma Hewitt - Be Your Sound (Official Music Video) : FxMtO7HvXzA
  Dash Berlin feat Jonathan Mendelsohn - Better Half Of Me (Acoustic) [Official Music Video] : De3Cjo23dOs
  Gareth Emery feat Christina Novelli - Dynamite : N2rj3os3IYg
  Giuseppe Ottaviani feat Amba Shepherd - Lost For Words (Official Music Video) : 94CwUWWEgxw
  Dash Berlin ft Kate Walsh - When You Were Around (Official Music Video) : LlGtQyOv2WA
  Dash Berlin - Never Cry Again (Jorn van Deynhoven Radio Mix) (Official Music Video) [High Quality] : 5gLIVqA9dFg
  Gareth Emery feat Christina Novelli - Concrete Angel [Official Music Video] : 0dFz10R529g
  Roman Messer feat Christina Novelli - Frozen (Official Music Video) : LNJZSyl7RtE
  Rigby - Lighter (Official Video) : dxa47ePobPs
  Sub Focus - Tidal Wave ft Alpines : B8vlk1UR99k
  Knife Party - Fire Hive (Krewella Remix) [Dubstep] : iuTOmyGXW6o
  Luke Bond feat Roxanne Emery - On Fire [Official Music Video] : pNM7bjb9JYw
  Gareth Emery feat Bo Bruce - U (Official Video) : jF7qGaKKD4M
  Craig Connelly & Christina Novelli - Black Hole [Official Music Video] : WPVv-Jjb0wQ
  Armin van Buuren feat Trevor Guthrie - This Is What It Feels Like (Official Music Video) : BR_DFMUzX4E
  Armin van Buuren presents Gaia - J'ai Envie De Toi (Official Music Video) : 0FgKQdc96k4
  Armin van Buuren feat Fiora - Waiting For The Night (Official International Music Video) : dU26cGlmkRg
  Armin van Buuren vs Sophie Ellis-Bextor - Not Giving Up On Love (Dash Berlin 4AM Mix) Official Video : MQKt1RoUB9c
}

Christian {
  NEEDTOBREATHE - 'Multiplied' (Official Video) : fGF-MGGLpB0
  NEEDTOBREATHE - 'Multiplied' (Live Acoustic Video) : vWT4gxAFQBs
  Sanctus Real - Lay It Down (Lyric Video) : q3f9GrbOXSA
  Chris Tomlin - Whom Shall I Fear [God of Angel Armies] [Lyrics] : qOkImV2cJDg
  Chris Tomlin - I Lift My Hands : c24En0r-lXg
  Chris Tomlin - How Great is Our God : -20GqU9Q4UE
  Chris Tomlin - I Will Rise (Live) : CKRF8UihM5s
  Audrey Assad - Breaking You (Live) : RYuGuxr7MB0
  Audrey Assad - Death Be Not Proud : sikF_phQHZw
  Audrey Assad - I Shall Not Want : e5xEYgGr6ms
  Dave Barnes - God Gave Me You : -zkrQMjlD3A
  Danny Gokey - Hope In Front of Me : O5GFiDdGGGM
  Among The Thirsty - Completely : sFC-d9PTusw
  7eventh Time Down 'Just Say Jesus' : T8CLgiYZyZE
  Switchfoot -- Love Alone Is Worth The Fight [Official Video] : rk9Pj3ID0UE
  Kari Jobe - I Am Not Alone (Lyric Video/Live) : I2oel0_Xa54
  Crowder - Come As You Are (Music Video) : r2zhf2mqEMI
  Francesca Battistelli - He Knows My Name (Official Audio) : 1NHQJWdXfFE
  Matt Redman - You Never Let Go (Passion '06) : 76ifTTuL4XI
  Lauren Daigle - How Can It Be (Lyric Video) : 6UXn_OuJkvE
  Francesca Battistelli - Write Your Story (Official Audio) : ecV1NHmELuA
  TobyMac - Speak Life : ZeBv9r92VQ0
  Luminate - Banner of Love (Lyric Video) : z4xk0dcc6gY
  Chris Tomlin - Awake My Soul (with Lecrae) [Lyrics] : fWpvknKuYrg
  Big Daddy Weave - Redeemed (Official Music Video) : VzGAYNKDyIU
  Crowder - I Am (Lyric Video) : cH_LLGiE0f0
  Crowder - Come As You Are | Live at the Grand Ole Opry | Opry : OuX9oTGBCw8
  Chris Tomlin - Jesus Messiah (Lyrics And Chords) : tdxSC1tHJn0
  Chris Tomlin - How Great Is Our God (Lyrics And Chords) : KBD18rsVJHk
}

misc {
  Ong Namo by Snatam Kaur : c1XCS0g6J4A
  Boyce Avenue - Speed Limit (Acoustic) : i9PxU3ZftmI
  Boyce Avenue - On My Way : Ie9VxRF7ucM
  One Republic - Apologize : fm0T7_SGee4
  Varun Rajput - State of Grace (Liquid Tension Experiment Cover) : 2bVNP9vd4FU
  Eric Stanley ft Kyle Landry - Inception 'Time' (Hans Zimmer Cover) : NUtvWyJ0F_o
  Yiruma - River Flows in You : 7maJOI3QMu0
  Howard Jones - No One Is To Blame : ENB2eX-U3a8
  Birdy - Wings (acoustic version) : 7tz1eYadQVY
  The Fighter Remix - Kait Weston (feat Saint Maurice and Eppic) : ijRVrGSaTms
  Robyn - Call Your Girlfriend - cover by Kait Weston Ft Sean Scanlon : VAlgkvKTOZ0
  BoB - Paper Route [Official Video] : t99LQ3zo8DU
  You Raise Me Up Violin Cover - Josh Groban - Daniel Jang : Z-zvV5Bsshc
  Thirty Seconds To Mars - Kings and Queens : onMdBNeGiK4
  Ludovico Einaudi - Nuvole Bianche [HD] : kcihcYEOeic
  PJ Simas - Show Goes On Remix : vAhjWd4_LZ0
  The Weeknd - Earned It (Fifty Shades Of Grey) (Lyric Video) : xe_iCkFsQKE
  Metallica - Enter Sandman Live Moscow 1991 HD : _W7wqQwa-TU
  Def Leppard - Adrenalize - 1992 (FULL ALBUM) : BTs98_pjwL4
  Metallica Fade to Black in real HD !!!! awesome !!!! : 0FMfsT11pdA
  Fun - We Are Young ft Janelle Monae [OFFICIAL VIDEO] : Sv6dMFF_yts
  30 Seconds To Mars - Closer To The Edge : mLqHDhF-O28
  Thirty Seconds To Mars - This Is War : Zcps2fJKuAI
  Fun - Some Nights [OFFICIAL VIDEO] : qQkBeOisNM0
  Fun - Carry On [OFFICIAL VIDEO] : q7yCLn-O-Y0
  Rachel Platten - Fight Song (Official Video) : xo1VInw-SKc
  Dark Fusion ft Amy Kirkpatrick - I Just Close My Eyes (Sunset Remix) [TranceFamily] Promo Video : avuoLKAi718
  UDM - New Sunrise (Radio Edit) [As Played on Uplifting Only 124] : kC44uGao2es
  Andy Blueman - Reflections (Original 2004 Mix) : fB8u-vUNZhQ
  Mark Eteson feat Audrey Gallagher - Breathe On My Own [Official Music Video] : jwO51I9zlaE
  Ace of Base - The Sign (Official) : iqu132vTl5Y
  Third Day - Cry Out To Jesus : JmVxRl5bc4Y
  Bob Cartel feat Ai Takekawa - Lie To Me (Akku Remix) [Just Music Records] Promo : ZmPlAHxalfw
  Colleen D'Agostino ft deadmau5 - Stay (Drop The Poptart Edit) : YPkwDy9okws
  3 Hours of Best Female Vocal Dubstep Mix May 2015 : SPgrxCHpyg8
  Dark Matters feat Jess Morgan - The Perfect Lie (Beat Service Remix) : dGd-QFuEUTA
  Alex H feat Mona Moua - There's No Turning Back (Original Mix) : 9vYULN-72Oo
  Ferry Corsten feat Betsie Larkin - Not Coming Down (Dash Berlin 4AM Remix) : A-gljGbLt-A
  Arnej feat Sally Saifi - Free Of You (8 Wonders Mix) : yr-kJ5PgAN4
  AMR feat Ai Takekawa - Beyond The Moon (Orbion Uplifting Remix) : liF-EZ4CFOw
  The Chainsmokers - Let You Go ft Great Good Fine Ok : iiNXf0n_hrA
  Inception - Time - Piano Solo HD : JOtDS0vRwxg
  Betsie Larkin & Solarstone  - Breathe You In (Solarstone Pure Mix) : 7-LpBG809Po
  Ferry Corsten presents Gouryella - Anahera [Official Music Video] : 7ZMZHbAKvGA
  Lqd Hrmny - La Distance : DhqYIyvnAvA
  Aaron Lewis - What Hurts The Most HD Live in Lake Tahoe 8/06/2011 : Yk9ByJZ0vrQ
  Taylor Swift - Style : -CmadmM5cOk
  Farhad Mahdavi - Yearning [As Played on Uplifting Only 126] : GqTM7eKZITg
  DEF LEPPARD - Long Long Way To Go (Official Music Video) : E2KTcr1Mir0
  Dash Berlin - #MusicIsLife - Album : DSOEUj8FAQ4
  Dash Berlin ft Sarah Howells - Go It Alone (Andrew Rayel Remix)(Official Music Video) : eWaANlb3q68
  illitheas - Endless (Final Version) [As Played on Uplifting Only 113] : kqnOnilA6ac
  Cinderella - Long Cold Winter - 1988 (FULL ALBUM) : BldJ5ejRKqw
  Slaughter - Stick It To Ya  (Full Album) : 5ZZvO3uBY-I
  Ai Takekawa & Adam Sobiech - Break The Light (Ryo Nakamura Remix) : N7u4wRwRQFM
  Denis Sender - Liquid Dreams (Ellez Ria Remix) : 2Vjjx72CVSY
  Annie Lennox - Why (Official Music Video) : HG7I4oniOyA
  AMR featAi Takekawa - Beyond The Moon - (Orbion Uplifting Remix) [HD] : 8tpvAptO7Vk
  Break Every Chain by Jesus Culture Lyrics : EtyVdC7E6Wo
  Syntouch - Lost Coast (Original Mix) *Promo* [Trancer Recordings] : jj8EkplRBqk
  Aeden - Tempest Sonata (Six Senses Remix) [TFB Records] [EXCLUSIVE PREMIERE] : Y4pg6v74C5s
  Hozier - Take Me To Church : PVjiKRfKpPI
  Metallica Live In Russia 1991 HQ - Monsters Of Rock Moscow : WJkeTEIUJhY
  Matt Maher - Lord I Need You (feat Audrey Assad) - Acoustic : iaVPupbNFAo
  Francesca Battistelli - He Knows My Name (Official Video) : jYpBgJHmGmw
  Usher - Yeah! ft Lil Jon Ludacris : GxBSyx85Kp8
  Come Alive (Dry Bones) featuring Lauren Daigle - Live from the Centric Worship Retreat : 7XAeyFagceQ
  Sunlight Project feat Danny Claire - Stay (tranzLift Remix) Promo : yWQvWTM7Hqg
  Simon O'Shine & Sergey Nevone - Last Goodbye (Original Mix) : QLqbwLPE-LY
  Chopin - The 21 Nocturnes (reference recording Claudio Arrau) : uUdoxvigIl8
}
```

**Comment by frankhale** [comment]
Been sitting on this for ages, this is happening with next commit. The entire project has been completely rewritten.

**Comment by frankhale** [comment]
Closed via a3871f846ec643de99dc510c98afd16141d2368c
